### DISCUSSION TITLE
"euronics" store fixed and updated

### DISCUSSION THREAD

**Discussion by ClaudioLiverano** [jef/streetmerchant#2921] [Show and tell]
_My first time moving on github, I don't know if this is the right section to post this._

I guys,

Euronics (.it) website is updated.. so I updated store/model/euronics.ts too for the inStock and outOfStock inputs.

`import {Store} from './store';

export const Euronics: Store = {
  currency: '€',
  labels: {
    inStock: {
      container: '.add-to-cart',
      text: ['Aggiungi'],
    },
	outOfStock: {
      container: '.out-of-stock',
      text: ['Puoi cercarlo nel tuo negozio preferito e prenotare il ritiro.'],
    },
  },
  links: [
    {
      brand: 'sony',
      model: 'ps5 console',
      series: 'sonyps5c',
      url: 'https://www.euronics.it/console/sony-computer/playstation-5/eProd202008906',
    },
    {
      brand: 'sony',
      model: 'ps5 digital',
      series: 'sonyps5de',
      url: 'https://www.euronics.it/console/sony-computer/playstation-5-digital-edition/eProd202008907',
    },
    {
      brand: 'microsoft',
      model: 'xbox series x',
      series: 'xboxsx',
      url: 'https://www.euronics.it/console/microsoft/xbox-series-x-1tb-it-italy-sxto/eProd202008981',
    },
    {
      brand: 'microsoft',
      model: 'xbox series s',
      series: 'xboxss',
      url: 'https://www.euronics.it/console/microsoft/xbox-series-s-512gb-it-italy-ltsn/eProd202008982',
    },
	{
      brand: 'msi',
      model: 'gaming x trio',
      series: '3090',
      url: 'https://www.euronics.it/informatica/212011012.html',
    },
  ],
  name: 'euronics',
};
`

**Comment by ClaudioLiverano** [comment]
Added Price Selector:

`maxPrice: {
      container: '.pcs-price',
      euroFormat: true,
    },`

**Comment by Mr-Tech-13** [comment]
You can submit a pr
